<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(218, 233, 255)</string>
  </property>
  <widget class="QLabel" name="photo_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>사진이 들어갈 라벨</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가격이 들어갈 라벨</string>
   </property>
  </widget>
  <widget class="QPushButton" name="name_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>이름이 들어갈 라벨</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
